scripts/prepare_disk.sh

```diff
@@ -22,6 +22,52 @@ usage()
     echo "NOTE: This script is destructive and will destroy your data."
 }
 
+is_resize_needed()
+{
+    is_comment()
+    {
+        test -z "${1%%#*}"
+    }
+
+    # Check if value is an integer, by using an inverse integer comparison test.
```

Comment:
actually, it's not inverse anymore. before we had to test if it was NOT an integer to be able to skip the line, now we actually test if it IS an integer. So we now just do an integer comparision test.
anyway, with the helper function, i'd argue ti's quite self explanatory? 1 = 1 :)